src/shared/components/rightbar/RightBar.tsx

```diff
@@ -44,6 +42,36 @@ export default class RightBar extends React.Component<
         );
     }
 
+    public newsContent = remoteData(async () => {
+        await sleep(5000);
+
+        return fetch(
+            'https://www.cbioportal.org/proxy/docs.cbioportal.org/news/'
```

Comment:
@inodb this will show on all portals except where there is What's New content specified in configuration.  Basically it will function exactly as Twitter feed did.  Is this what we want?

Reply:
good question - i guess maybe people may want to point to some custom markdown?

For e.g. GENIE, we could point to genie news: https://docs.cbioportal.org/news-genie/

Reply:
The other thing is that if somebody is running an old portal, not sure what a good default would be. I guess if the news markdown is part of the project we could link to that directly rather than the cbioportal.org news page?

Reply:
we agreed to just deploy this.  it will function exactly as twitter embed did.